sdcm/utils/k8s.py

```diff
@@ -89,14 +89,31 @@ def run(self) -> None:
             time.sleep(1 / self.rate_limit)
 
     def wrap_api_client(self, client):
-        orig_call_api = client._ApiClient__call_api  # deepcode ignore W0212: monkey patching in here
-
-        @wraps(orig_call_api)
-        def call_api(*args, **kwargs):
-            self.wait("k8s API call: args=%s, kwargs=%s", args, kwargs)
-            return orig_call_api(*args, **kwargs)
-
-        client._ApiClient__call_api = call_api  # deepcode ignore W0212: monkey patching in here
+        # deepcode ignore W0212: monkey patching in here
+        orig_connection_from_host = client.rest_client.pool_manager.__class__.connection_from_host
+        orig_pm_urlopen = client.rest_client.pool_manager.__class__.urlopen
+
+        @wraps(orig_pm_urlopen)
+        def pm_urlopen(*args1, **kwargs1):
+            kwargs1['retries'] = 2
+            return orig_pm_urlopen(*args1, **kwargs1)
+
```

Comment:
Number of retries is configurable thru k8s `Configuration` object: https://github.com/kubernetes-client/python/blob/eb9b6d609e44838f4ee04477d9908d581481ec72/kubernetes/client/configuration.py#L165. Don't need to change it this way.

Reply:
This is recent change, does not preset in version we are currently using.

Reply:
So, we need to bump kubernetes-py library version?

Reply:
done, no monkeypatching anymore